.github/workflows/smoke-test.yml

```diff
@@ -0,0 +1,23 @@
+name: Smoke test
+
+on:
+  schedule:
+    - cron: '0 0 * * SUN'
+  workflow_dispatch:
```

Comment:
can we make sure this only runs on the main repo, and not on everyone's fork?

Reply:
The `on.schedule` does not support similar `branches` option as `push|pull_request` but it can be limited by checking the repository from github context. 

```yml
jobs:
  lint:
    if: ${{ github.repository == 'yannickcr/eslint-plugin-react' }}
    runs-on: ubuntu-latest
```

Example run here https://github.com/AriPerkkio/eslint-plugin-react/runs/2477243324?check_suite_focus=true